traits/constants.py

```diff
@@ -10,6 +10,19 @@
 
 from enum import IntEnum
 
+from traits.ctraits import (
```

Comment:
This introduces a circular import: `traits.trait_base` imports from `traits.constants`, which imports from `traits.ctraits`, which imports from `traits.trait_base`. The solution may be to lift `CoercableTypes` to `trait_handlers`, which is the only place that it's used.

Reply:
Or perhaps put `Undefined` and `Uninitialized` somewhere other than in `trait_base`?